docs/index.rst

```diff
@@ -13,12 +13,12 @@ Arrow is heavily inspired by `moment.js <https://github.com/timrwood/moment>`_ a
 ----
 Why?
 ----
-Python's standard library and some other low-level modules have near-complete date, time and time zone functionality but don't work very well from a usability perspective:
+Python's standard library and some other low-level modules have near-complete date, time and timezone functionality but don't work very well from a usability perspective:
```

Comment:
There was inconsistent use of both "time zone" and "timezone".  I personally prefer the latter, but I can change this.  (Even Python's own docs are inconsistent with this spelling.)